docroot/profiles/custom/cgov_site/modules/custom/cgov_schema_org/src/Plugin/Field/FieldType/CgovSchemaOrgFieldItem.php

```diff
@@ -56,8 +56,7 @@ public static function propertyDefinitions(FieldStorageDefinitionInterface $fiel
    * {@inheritdoc}
    */
   public function isEmpty() {
-    $page_itemtype = $this->get('page_itemtype');
-    return empty($page_itemtype);
+    return empty($this->get('page_itemtype')) ? TRUE : FALSE;
```

Comment:
I mean, empty is a function itself that returns a bool. So why the need for the ternary?

Reply:
phpstan throwing an error and because of that i just added that way.